lecture_notes/09 functions/3 - default.py

```diff
@@ -0,0 +1,8 @@
+def max(a=0, b=0):
+    if a > b:
+        return a
+    else:
+        return b
```

Comment:
Use ternary operator and/or lambda syntax

Reply:
Done!